src/universal/modules/meeting/components/MeetingLayout/MeetingLayout.js

```diff
@@ -17,6 +17,7 @@ MeetingLayout.propTypes = {
 styles = StyleSheet.create({
   root: {
     backgroundColor: '#fff',
+    content: '"MeetingLayout"',
```

Comment:
Hehe, was tracking down styles:component. Maybe I should be using a React DOM inspector? I can clean these up at any rate.